core/templates/dev/head/components/state-directives/answer-group-editor/answer-group-editor.directive.ts

```diff
@@ -137,7 +128,8 @@ angular.module('oppia').directive('answerGroupEditor', [
 
                   $scope.selectMisconception = function(
                       misconception, skillId) {
-                    $scope.selectedMisconception = angular.copy(misconception);
+                    $scope.selectedMisconception =
+                    angular.copy(misconception);
```

Comment:
Why this change? Also, indentation is wrong.

Ditto below, too.

Reply:
Fixed.